app/Router.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { ApolloClient, InMemoryCache, createHttpLink } from '@apollo/client';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Link,
+  Route
+} from "react-router-dom";
+import routes from "../config/routes";
+import store from "../lib/store";
+import Controller from "../lib/Controller";
+import { setDiamondhandsClients } from "../lib/ApolloController";
+
+
+class Router extends React.Component {
```

Comment:
This file will only exist in beat of the day or hey robot, but not in the diamond hands library, but we can provide an example of you yours should look